docs/index.rst

```diff
@@ -1,159 +1,325 @@
 2. minimum-tokens - minimum set of tokens that are needed for each of your
    tests.  Default minimum tokens are ``assert``, ``feature`` and ``test``.
 
-In a test case docstring, every token should be prefixed with ``@`` and
-suffixed with ``:``. This will ensure that Testimony provides right reports.
-The reports supported by testimony are:
+.. note::
 
-1. print - prints all captured tests with the parsed tokens for each test. Also
-   it prints non-recognized tokens.
-2. summary - presents a summary of all tests with useful information like:
-   number of tests missing docstring, number of tests corresponding to each
-   defined token, and total number of test cases.
-3. validate - helps ensure that all your tests atleast have the minimal set
-   of tokens defined.  The information presented by this command will help you
-   identify the issues pertaining to each test.  A non-zero return code will be
-   returned when:
+  To help test case parsing, make sure that each test case docstring has the
+  tokens prefixed with ``@`` and suffixed with ``:``.  Otherwise, you may see
+  incorrect results.
 
-     - minimal set of tokens is not present for atleast 1 test case
-     - one or more tests is missing the docstring
-     - An unexpected token identified
+.. _sample_testcase:
 
-Advantages
-----------
-1. Avoids using a test case management tool to document test cases by reusing
-   the Python automation framework.
-2. Enforces test cases to follow a defined standard.
-3. Runs in integration with tools like Travis to get a report as and when you
-   check in the code.
-4. Saves a lot of time from the conventional way of writing test cases using a
-   test management tool.
-5. The test case information can be easily extracted from Testimony and can be
-   ported to any test management tool.
+Sample Test Case
+++++++++++++++++
+A sample python test case with test case tokens is shown below:
 
-Test lookup and definition
---------------------------
+.. code-block:: python
 
-The ``PATH`` argument of testimony accepts more than one value.  Testimony
-captures all python test modules (``.py`` files whose names start with
-``test_``) in the given paths. If ``PATH`` is a directory, then the directory
-and its subdirectories will be considered as well.
+    def test_login_1(self):
+    """Check if a user is able to login with valid userid and password
 
-Inside a test module, Testimony looks for functions whose names start with
-``test_``.  It then parses the function docstrings and extracts the tokens.
-Also it creates namespaces for module and class level docstrings which will
-then be reused in the children tests. For example, if a module has a token
-called ``feature``, then all tests in that module will inherit it by default.
-But the individual tests can choose to override this value by defining their
-own.  The token lookup will happen in the following order and it will stop on
-the very first match::
+       More description for the test.
```

Comment:
This and the following lines should be unindented to align with the column where `"""` starts.